charts/incubator/cryptpad/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+appVersion: v4.12.0-nginx
```

Comment:
Cryptpad provides images with and without a built-in nginx reverse proxy.

In my first approach I attempted to use the one without the built-in  nginx, to avoid having an "extra" reverse proxy behind the ingress controller. However, this turned out to be very hard: Cryptpad requires a very specific config with two domains and a bunch of different endpoints for static data and different rules for setting headers.

Having two reverse proxies is not the best but it is definitely more maintainable.

Reply:
This is pretty much like any other application in which a programming language doesn't have a built in web server. Using nginx is fine with me